velox/connectors/hive/storage_adapters/gcs/examples/GCSFileSystemExampleRead.cpp

```diff
@@ -13,6 +13,7 @@
  * See the License for the specific language governing permissions and
  * limitations under the License.
  */
+#include "velox/common/memory/Memory.h"
```

Comment:
Why do we need this header?

Reply:
Copy and paste is a destructive drug, don't consume drugs!